File: src/main/java/com/flowcommerce/core/support/auth/PasswordEncoder.java
package com.flowcommerce.core.support.auth;

import lombok.NoArgsConstructor;
import org.springframework.stereotype.Component;

@Component
@NoArgsConstructor
public class PasswordEncoder {

    // TODO 암호화 알고리즘 적용해야함
    public String encode(String password) {
        return password;
    }

    public boolean matches(String password, String encodedPassword) {
        return password.equals(encodedPassword);
    }
}

File: src/main/kotlin/com/flowcommerce/core/support/auth/PasswordEncoder.kt
package com.flowcommerce.core.support.auth

import org.springframework.stereotype.Component

@Component
class PasswordEncoder {

    // TODO 암호화 알고리즘 적용해야함
    fun encode(password: String): String = password

    fun matches(password: String, encodedPassword: String): Boolean =
        password == encodedPassword
}
